hack/packages/check-sample-values-and-render-ytt.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# this shell script checks the existence of a sample values file for a Tanzu package and verifies if ytt templates are generated.
+
+# set this value to your package name
+PACKAGE=$1
+
+# set this value to the version of your package
+VERSION=$2
+
+if [ -z "$PACKAGE" ]
+then
+  echo "check sample values for package failed. must set NAME"
+  exit 2
+fi
+
+if [ -z "$VERSION" ]
+then
+  echo "check sample values for package failed. must set VERSION"
+  exit 2
+fi
+
+ROOT_DIR="addons/packages"
+PACKAGE_DIR="${ROOT_DIR}/${PACKAGE}"
+VERSION_DIR="${PACKAGE_DIR}/${VERSION}"
+BUNDLE_DIR="${VERSION_DIR}/bundle"
+CONFIG_DIR="${BUNDLE_DIR}/config"
+NC='\033[0m' # No Color
+GREEN='\033[0;32m'
+
+check_sample_values_and_render_ytt() {
+  sample_values_dir="${BUNDLE_DIR}/sample-values"
```

Comment:
Just realized that if `sample-values` is put inside `bundle`, it will be pushed as package content when we execute `make push-package`. It is something we desire? If not, we need to move `sample-values` a level up. @shivaani0505 @vijaykatam

Reply:
yeah I missed it as well, `sample-values` shouldn't be part of the bundle.

Reply:
will move this one level up